scripts/helpers.js

```diff
@@ -3,7 +3,10 @@ const path = require('path');
 const xa = require('xa');
 
 const compStateLogger = function (componentName, text) {
-	xa.custom(componentName.toUpperCase(), text, { titleColor: 'yellow', backgroundColor: '#212121' });
+	const debugMode = false;
+	if (debugMode) {
+		xa.custom(componentName.toUpperCase(), text, { titleColor: 'yellow', backgroundColor: '#212121' });
+	}
```

Comment:
Interesting way to comment out a line :)

Reply:
It was suppose to be temporary until I replace it with an env variable :D I'll change it.